.ci/scripts/build_llama_android.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -ex
+
+install_executorch_and_backend_lib() {
+  echo "Installing libexecutorch.a, libportable_kernels.a, libetdump.a, libbundled_program.a"
+  rm -rf cmake-android-out && mkdir cmake-android-out
+  ANDROID_NDK=/opt/ndk
+  BUCK2=$(which buck2)
+  FLATC=$(which flatc) ANDROID_ABI=arm64-v8a
+  cmake -DBUCK2="$BUCK" -DCMAKE_TOOLCHAIN_FILE="$ANDROID_NDK"/build/cmake/android.toolchain.cmake -DANDROID_ABI=arm64-v8a -DANDROID_PLATFORM=android-23 -DCMAKE_INSTALL_PREFIX=cmake-android-out -DCMAKE_BUILD_TYPE=Release -DEXECUTORCH_BUILD_EXTENSION_MODULE=ON -DEXECUTORCH_BUILD_EXTENSION_DATA_LOADER=ON -DEXECUTORCH_ENABLE_LOGGING=1 -DEXECUTORCH_BUILD_XNNPACK=ON -DPYTHON_EXECUTABLE=python -DEXECUTORCH_BUILD_OPTIMIZED=ON -DXNNPACK_ENABLE_ARM_BF16=OFF -Bcmake-android-out .
```

Comment:
nit format one option per line? Just too hard to read / parse - something like this

```suggestion
cmake                                                                         \
    -DBUCK2="$BUCK"                                                           \
    -DCMAKE_TOOLCHAIN_FILE="$ANDROID_NDK"/build/cmake/android.toolchain.cmake \
    -DANDROID_ABI=arm64-v8a                                                   \
    -DANDROID_PLATFORM=android-23                                             \
    -DCMAKE_INSTALL_PREFIX=cmake-android-out                                  \
    -DCMAKE_BUILD_TYPE=Release                                                \
    -DEXECUTORCH_BUILD_EXTENSION_MODULE=ON                                    \
    -DEXECUTORCH_BUILD_EXTENSION_DATA_LOADER=ON                               \
    -DEXECUTORCH_ENABLE_LOGGING=1                                             \
    -DEXECUTORCH_BUILD_XNNPACK=ON                                             \
    -DPYTHON_EXECUTABLE=python                                                \
    -DEXECUTORCH_BUILD_OPTIMIZED=ON                                           \
    -DXNNPACK_ENABLE_ARM_BF16=OFF                                             \
    -Bcmake-android-out .
```

Reply:
why do we have to disable BF16? Which version of the NDK we have for the CI job?

Reply:
r26c, I ran into error with DXNNPACK_ENABLE_ARM_BF16=OFF, maybe can try removing it later

Reply:
yeah, I didnt have to do this. I would suggest try removing it